with-custom-font/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { useFonts } from '@use-expo/font';
+import { Text, View, StyleSheet } from 'react-native';
+import { AppLoading } from 'expo';
+
+let customFonts = {
+  'Inter-Black': require('./assets/fonts/Inter-Black.otf'),
+  'Inter-SemiBoldItalic': 'https://rsms.me/inter/font-files/Inter-SemiBoldItalic.otf?v=3.12',
```

Comment:
Do we also want to [showcase the fine-grained control for webfonts](https://docs.expo.io/versions/latest/sdk/font/#fontdisplay)? I recently stumbled on it by accident, could be cool for other people to show off what `expo-font` is capable of? 😄

Reply:
sure let's do it